apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -3,16 +3,20 @@ import { getValidClassNames } from "~/libs/helpers/helpers.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	children?: React.ReactNode;
 	isFluid?: boolean;
+	isIconButton?: boolean;
```

Comment:
I don't think we need that. If we have icon, then it is icon button. Add `iconName` prop here and avoid `children`